sammba/registration/struct.py

```diff
@@ -167,11 +169,13 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
     # and set the NIfTI image centre (as defined in the header) to the CoM
     # of the extracted brain.
     if unifize_kwargs is None:
-        unifize_kwargs = {'cl_frac': .2}
+        unifize_kwargs = {}
 
     unifized_files = []
     for n, anat_file in enumerate(copied_anat_filenames):
-        out_unifize = unifize(in_file=anat_file, outputtype='NIFTI_GZ',
+        out_unifize = unifize(in_file=anat_file,
+                              out_file='%s_Unifized_for_brain_extraction.nii.gz',
```

Comment:
isn't this one for the registration and this other for brain extraction ?

Reply:
strictly speaking it's for registration *and* brain extraction. the first/RATS_MM step is for brain *masking*. brain extraction can be said to have happened if it is then done with a mask

Reply:
Sorry I am not sure to follow. Doesn’t brain extraction means putting to zero voxels that are not part of the brain ?

Reply:
yes, and that's done on the second file. the mask used to do it with come from the first

Reply:
Ah yes you are right. So actually this is a little bit a mess due to the names of the kwargs. The thing is that one unifization is just useful for the computation of the brain mask, and the other one used for all the remaining steps, meaning also the output will have this unifization amount. so may be the `brain_extraction_unifize_kwargs` should be renamed to `brain_masking_unifize_kwargs` and then the output files following the associated kwargs names, that is **%s_unifized_for_brain_masking** for `brain_masking_unifize_kwargs` and keep default name for `unifize_kwargs`